src/Components/App/App.js

```diff
@@ -11,21 +11,23 @@ import './App.css';
 
 const App = () => {
   const [user, setUser] = useState({id: 1, name: 'KB'});
-  const [userLogs, setUserLogs] = useState(entryData);
+  const [userLogs, setUserLogs] = useState([]);
   const [feeling, setFeeling] = useState('');
   const [activity, setActivity] = useState('');
   const [journal, setJournal] = useState('');
   const [activities, setActivities] = useState(activityData);
   const [filteredActivities, setFilteredActivities] = useState([]);
-
+  useEffect(() => {
+    setUserLogs(entryData);
+  }, []);
```

Comment:
Love the setup to transition to fetching easily! Could potentially add `user` as a dependency for future-proofing - this would update the logs any time the user changed!

Reply:
oh REALLY?? I should have mentioned that, i didn't wanna just implement it without chatting with y'all. It would not have been ideal for me to just use it because if some of the class names were the same as bootstraps (in any file) those styles would have overridden them!

Reply:
Smart use of useEffect. Like Joe said, it will definitely ease the transition to fetching data.